### ISSUE TITLE
Hyperlinks as references

### ISSUE THREAD

**Issue by muralidn** [liziscool/cas741_project#10]
@liziscool 

All the hyperlinks (including that of Wikipedia) in the TM, GD, DD, and IM should be replaced with citations.

**Comment by smiths** [comment]
Yes, citations are better than hyperlinks.  @liziscool, please include the commit hash when you close this issue.
